README.md

```diff
@@ -467,6 +467,20 @@ stacks:
       template: my-stack.rb
 ```
 
+### Loading SparklePacks
+
+[SparklePacks](http://www.sparkleformation.io/docs/sparkle_formation/sparkle-packs.html) can be pre-loaded using compiler options. This requires the name of a rubygem to `require` followed by the name of the SparklePack, which is usually the same name as the Gem.
```

Comment:
Might be worth stating that we are deviating from SparkleFormation prescribed technique. 
i.e. You wont be doing this

```
require 'my_sparkle_pack'

sfn = SparkleFormation.new(:my_template) do
   dynamic!(:my_dynamic)
end
sfn.sparkle.add_sparkle(custom_pack)
```
It's just
```
SparkleFormation.new(:my_template) do
   dynamic!(:my_dynamic)
end
```

Reply:
Added in 978dfea